src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Hmm, I never had need for BEM syntax, maybe fml. I suppose it's over scope of Este.js and way too opinioned. Also, it will not be compatible with future React CSS. Can you please debemify it? Then I will hapily merge the PR. Thank you!